### DISCUSSION TITLE
CVSD

### DISCUSSION THREAD

**Discussion by ahlawy81** [jgaeddert/liquid-dsp#341] [General]
Hello, thanks for your efforts
I wonder if you are familiar with CVSD vocoder and FSK demodulation

**Comment by jgaeddert** [comment]
I am with both. Liquid includes implementations for both CVSD and FSK demodulation.

**Reply by ahlawy81** [reply]
Could you please tell me how could it help with CVSD coded signal with fsk modulation decoding
